tests/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+#=============================================================================
```

Comment:
is this file necessary? if you meant to add this file, then I feel you should also update this line: https://github.com/nv-legate/legate.core/blob/branch-23.07/legate_core_cpp.cmake#L521